docs/textsearch.adoc

```diff
@@ -182,4 +182,18 @@ mgmt.buildIndex('name', Vertex.class).addKey(string, Mapping.STRING.asParameter(
 mgmt.buildIndex('document', Vertex.class).addKey(text, Mapping.TEXT.asParameter()).buildMixedIndex("search")
 mgmt.commit()
 
-With these settings, JanusGraph will use a SimpleAnalyzer analyzer for property key 'string', an EnglishAnalyzer analyzer for property key 'text',  a KeywordAnalyzer analyzer for property 'name' and a StandardAnalyzer analyzer for property 'document'.
\ No newline at end of file
+With these settings, JanusGraph will use a SimpleAnalyzer analyzer for property key 'string', an EnglishAnalyzer analyzer for property key 'text',  a KeywordAnalyzer analyzer for property 'name' and a StandardAnalyzer analyzer for property 'document'.
+
+==== Custom parameters
+
+Some index backends are supporting custom parameters (right now only Elasticsearch supports custom parameters). The name of the custom parameter must be set through ParameterType.customParameterName("yourProperty").
```

Comment:
Could you add a short explanation for the custom parameters to describe why they are useful?